.nuget/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NuGet.CommandLine" version="2.1.0" />

```

Comment:
Not a must. Updating the Common.Logging package is currently the breaking issue.

Hopefully, you could help us and other users by accepting those related commits (or by updating the package yourself) and publishing an updated NuGet package to the repository (we had to do a custom build to avoid the incompatibilities problems). Thanks!